flake.nix

```diff
@@ -3,7 +3,7 @@
     common.url = "github:nammayatri/common";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/9e29a21ac28ecbcd2e63db79fa547d7abc935143";
```

Comment:
@srid will change it back to pointing to main, raised PR with this change for getting it reviewed (and getting green tick from biryani 🙈)

Reply:
Ah, Okay.